### PR TITLE
Tf?

### PR DESCRIPTION
**Pull request by MajManatee**
# Document the changes in your pull request
Adds my reaction when I saw that headpatting PR to the pretty filter

# Changelog

:cl:  
tweak: TF > the fuck?
/:cl:

### CODE REVIEW

**Comment by Hislyna** [comment]
mfw all these new filters I use irl and now I feel this game be too unrealistic. who tf has time to talk like a normal human nowa days

**Comment by solwashere** [comment]
Manatee is atoning for their sins

File: config/minor_filter.txt [hunk 1]
```diff
@@ -41,4 +41,5 @@ hbu=how about you
 \bur\b=your
 btw=by the way
 pls=please
-plz=please
\ No newline at end of file
+plz=please
+tf=the fuck
```
Comment (Moltijoe) [inline config/minor_filter.txt]:
```suggestion
\btf\=the fuck
```

**Comment by MajManatee** [comment]
there is only one sinner on this github

File: config/minor_filter.txt (hunk 1 above)
Comment (Moltijoe) [reply config/minor_filter.txt]: i assume this makes it require it not being part of a word, since there's probably a word out there that uses "tf"

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: config/minor_filter.txt (hunk 1 above)
Comment (MajManatee) [reply config/minor_filter.txt]: its the inverse actually

Comment (Maia-J) [reply config/minor_filter.txt]: It isnt, \b \b makes it require spaces

**Comment by Maia-J** [comment]
> there is only one sinner on this github

UwU

File: config/minor_filter.txt (hunk 1 above)
Comment (Maia-J) [inline config/minor_filter.txt]:
```suggestion
\btf\b=the fuck
```

I double checked, Molti was right. This should make it not turn "outfit" into "outhe fuckit"

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Review by Moltijoe — APPROVED** [review]

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**